lib/checkSystem.js

```diff
@@ -111,7 +111,7 @@ let check = function(pathToPackage) {
                 commandError: error,
                 type: 'error'
             });
-            return Promise.reject();
+            return Promise.reject('');
```

Comment:
why are we now rejecting an empty string?

Reply:
@mohlsen I used xo linter and fulfilled its recommendations.